README.md

```diff
@@ -159,14 +160,27 @@ database format. This can be done by
 python manage.py makemigrations --settings=config.settings.local
 ```
 
+The migration might have to be improved, e.g. by using `migrations.RunPython`.
+
 Afterwards the test database has to be updated as well. Use the dumpdata command to generate a test database from the
 currently running database:
 
 ```shell
 python manage.py dumpdata --settings=config.settings.local > e2e_tests/database/test_database.json
 ```
 
-On Windows (and possibly others): Check the diff of `e2e_tests/database/test_database.json` for any unexpected parts and adjust as necessary.
+Cheat-sheet to make sure the correct data is dumped:
+
+```shell
+git checkout right-before-model-change
+rm db.sqlite3
+./manage.py migrate --settings=config.settings.local
+./manage.py loaddata --settings=config.settings.local e2e_tests/database/test_database.json
+git checkout after-model-change-including-migration
+./manage.py migrate --settings=config.settings.local
+```
+
+Check the diff of `e2e_tests/database/test_database.json` for any unexpected parts and adjust as necessary.
```

Comment:
I don't have a clue why this is necessary and why we cannot just dump the new state and what could be incorrect about the data being dumped. Maybe you could add a single line of explanation why we need this. But then again that might be just me cause you're in this way deeper than me.

Reply:
That line is not actually new. On Windows it gave me a single line, while the checked-in json was structured. And some umlauts were not correct... It might just be a Windows problem, but it doesnt really hurt to have a quick look at the diff.

Reply:
My comment refered to the whole block of text that was added mainly the cheat sheet the need of which is unclear to me.